recipe/meta.yaml

```diff
@@ -41,104 +44,107 @@ outputs:
         - cmake
         - make
         - llvm-openmp  # [osx]
+        - posix
```

Comment:
`posix` should be installed by default, so this shouldn't be necessary. Otherwise, if you really need it, try to keep the build requirements here and in the top-level recipe in sync.